### PR TITLE
Limit running multiple import build

### PR DESCRIPTION
**Pull request by jkciesluk**
Connected to https://github.com/scalameta/metals/issues/2151

Now we can't force multiple build import using `Import Build` command

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala [hunk 1]
```diff
@@ -170,7 +170,8 @@ class MetalsLanguageServer(
   private val recentlyOpenedFiles = new ActiveFiles(time)
   private val recentlyFocusedFiles = new ActiveFiles(time)
   private val languageClient = new DelegatingLanguageClient(NoopLanguageClient)
-
+  @volatile
+  private var isImportInProcess: Boolean = false
```
Comment (kpodsiad) [inline metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala:174]: If we might end up in situation when two or more threads can modify the same variable, shouldn't `AtomicRef` (in this case `AtomicBoolean`) and CAS loops be used? At least I'm always going with Atomic* whenever I had to share some mutable value.

Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala:174]: Ach yeah, we might need an AtomicBoolean or something. That's my bad, I suggested volotile

Comment (kpodsiad) [reply metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala]: Just an idea: maybe other usages of volatile should be rethinked?

**Review by tgodzik — APPROVED** [review]
LGTM!

**Pull request merged**